src/sentry/static/sentry/less/group-detail.less

```diff
@@ -1064,7 +1064,6 @@ span.val {
 }
 
 .val-string {
-  color: darken(@purple, 10);
```

Comment:
Do you need to change the colours of the other dumped content types? I'm thinking of numbers, null and boolean which also have colours defined.

Reply:
Ah most likely, good catch

Reply:
@markstory actually, I think this might be ok to keep around?

![image](https://user-images.githubusercontent.com/79684/102286408-1545cf80-3eed-11eb-840e-33719665acd6.png)

Reply:
![image](https://user-images.githubusercontent.com/79684/102286432-255daf00-3eed-11eb-860f-0766a6241bc1.png)